source/sync/data-model/update-schema.txt

```diff
@@ -1,441 +1,589 @@
+~~~~~~~~~~~~~
 
-You can add an object type to either the server-side schema or the client 
-object model without doing any additional handling. 
+Adding an object type is a non-breaking change when made to the server-side
+schema or the client-side SDK data model.
 
-If you want to add an object type to both the server-side schema and the 
-client object model, you can add the object type to the object model, and 
-use :ref:`Development Mode <development-mode>` to let App Services handle the 
-server-side schema updates. Or you can manually add the object type to both 
-the model and the schema.
+You can make your changes from the client or server side:
 
-.. note:: These changes may trigger a resync
+- **Client Side:** Use :ref:`Development Mode <development-mode>` to let App
+  Services handle the server-side schema updates.
+- **Server Side:** Manually add the object type to both the client model and the
+  schema.
 
-   When you add a new object type, we retrieve the documents for the 
-   collection and re-insert them into App Services to get the new values. This
-   is expected behavior, but it does cause a temporary halt to propagating 
-   changes while this process is underway.
+.. note:: May Cause Documents to Resync
 
-.. _schema-update-add-a-default-value:
-
-Add a Default Value on a Property
----------------------------------
+  If you have existing documents in Atlas and you add a new object type, the
+  server retrieves the documents in Atlas for the collection and re-inserts them
+  to get the new values. This is expected behavior, but it does temporarily halt
+  syncing changes until the process completes.
 
-You can add a default value on an object's required property. When you 
-insert an Atlas document missing this required property into the 
-collection, the Realm clients set the property to the default value. However, 
-the same property on the Atlas document remains empty until a client makes 
-changes to the property or updates the document directly in Atlas.
+.. _schema-update-add-a-default-value:
 
-For more information on how default values are helpful when modifying 
-existing Atlas documents, see :ref:`<schema-update-add-required-property>`.
+Set a Default Value on a Property
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. warning:: Ensure the default value type and property's type are the same
+Setting a default value on an existing object property is a non-breaking change
+when made to the server-side schema or the client-side SDK data model.
 
-   The default field does not have type validation. If the default field's type
-   and the property's type are not the same, the error will indicate that
-   the document is missing a required field. 
+We recommend that you always set a default value to a property whenever you add
+a new required property, either on its own or as a property for a new object
+type.
 
-.. _schema-update-add-required-property:
+You can make your changes from the client or server side:
```

Comment:
Setting a default value on the client doesn't impact the server's default value. Thus, users should ideally be updating the server's value